samples/azure/azure-service-fabric-routing/Core_6/CandidateVoteCount/EndpointCommunicationListener.cs

```diff
@@ -102,6 +103,14 @@ static void ConfigureSenderSideRoutingCandidateVoteCount(TransportExtensions<Azu
         #endregion
     }
 
+    public async Task Run()
+    {
+        if(endpointConfiguration != null)
```

Comment:
Protective code in case this method is called not from `service.RunAsync()`, but earlier. Arguably, it should check for null and throw an exception with explanation.